ODLA/platforms/dnnl/odla_dnnl_binary.cc

```diff
@@ -17,6 +17,7 @@
 
 #include <Eigen/Core>
 
+#include "Eigen/src/Core/util/Constants.h"
```

Comment:
Do we need this?
If so, we should make line 18  and line 20 the same style.

Reply:
My bad, deleted now